src/App.tsx

```diff
@@ -2,10 +2,10 @@ import React from "react";
 
 function App() {
   return (
-      <h1>
-        What to watch
-      </h1>
+    <>
+      
+    </>
```

Comment:
why it was changed?

Reply:
greeting block was on this place. now <h1> added just for test